tests/ft_calloc_test.cpp

```diff
@@ -21,6 +22,28 @@ int main(void)
 	/* 1 */ check(!memcmp(p, e, 4));
 	/* 2 */ mcheck(p, 4); free(p); showLeaks();
 	/* 3 */ check(ft_calloc(SIZE_MAX, SIZE_MAX) == NULL);
+
+	/* @evportel */
+	/* 4 */ check(ft_calloc(INT_MAX, INT_MAX) == NULL);
+	p = ft_calloc(INT_MIN, INT_MIN);
+	/* 5 */ check(p == NULL); free(p); //showLeaks();
+	p = ft_calloc(0, 0);
+	/* 6 */ check(p != NULL); free(p); //showLeaks();
+	p = ft_calloc(0, 5);
+	/* 7 */ check(p != NULL); free(p); //showLeaks();
+	p = ft_calloc(5, 0);
+	/* 8 */ check(p != NULL); free(p); //showLeaks();
+	p = ft_calloc(-5, -5);
+	/* 9 */ check(p == NULL); free(p); //showLeaks();
```

Comment:
When the function must return NULL there is no need to free.
IMO we should also showLeaks just in case.

```suggestion
	/* 9 */ check(ft_calloc(-5, -5) == NULL); showLeaks();
```